x/liquidstaking/keeper/tally.go

```diff
@@ -241,10 +241,8 @@ func (k Keeper) SetLiquidStakingVotingPowers(ctx sdk.Context, votes govtypes.Vot
 			continue
 		}
 
-		// if the denom is pool coin, calc btoken share and add owned btoken on bTokenOwnMap
-		bTokenSharePerPoolCoin := k.TokenSharePerPoolCoin(ctx, liquidBondDenom, denom)
-		if bTokenSharePerPoolCoin.IsPositive() {
-			bTokenSharePerPoolCoinMap[denom] = bTokenSharePerPoolCoin
+		// if the denom is pool coin, get bToken share and add owned bToken on bTokenOwnMap
+		if bTokenSharePerPoolCoin, ok := bTokenSharePerPoolCoinMap[denom]; ok && bTokenSharePerPoolCoin.IsPositive() {
```

Comment:
Is it necessary to validate `bTokenSharePerPoolCoin.IsPositive()` in this line?

Reply:
@kogisin I don't think so, since `GetBTokenSharePerPoolCoinMap` sets only positive shares.

https://github.com/cosmosquad-labs/squad/blob/1ce813826bc59066ccf04e81984dbf9e71a691bc/x/liquidstaking/keeper/tally.go#L38-L49

Reply:
We don't need to check again, Thanks let me remove the code